src/ape/api/transactions.py

```diff
@@ -40,7 +41,7 @@ class TransactionAPI(BaseInterfaceModel):
     # If left as None, will get set to the network's default required confirmations.
     required_confirmations: Optional[int] = Field(None, exclude=True)
 
-    signature: Optional[TransactionSignature] = Field(exclude=True)
+    signature: Optional[TransactionSignature]
```

Comment:
Why do we need the signature like this?

Reply:
Same as the previous comment. And I needed that signature data for the caching system.